cfme/tests/intelligence/chargeback/test_chargeback_long_term_rates.py

```diff
@@ -26,7 +27,7 @@
     pytest.mark.tier(2),
     pytest.mark.parametrize('interval', ['Daily', 'Weekly', 'Monthly'],
         ids=['daily_rate', 'weekly_rate', 'monthly_rate'], scope='module'),
-    pytest.mark.provider([VMwareProvider], selector=ONE,
+    pytest.mark.provider([RHEVMProvider], selector=ONE,
                        scope='module',
```

Comment:
I'll work on that change through another PR since the most of the other fixtures in my tests are module scoped , because of which there would be a scope mismatch.

These code changes should fix the jenkins failure.

Reply:
Talked to @nachandr and this is a bigger change as the provider is being accessed from module scoped fixtures .